### PR TITLE
Setup project step 1

### PR DESCRIPTION
**Pull request by Dachrono**
- Create index.html
- Create index.js
- Create style.css 
- Create modules

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Dachrono, Kindly fix lighthouse errors 👍🏿

**Comment by prolajumokeoni** [comment]
Hi @Dachrono 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has lighthouse linter errors, please fix it 👍🏿 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/form.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default function form() {
```
Comment (wayungi) [inline src/modules/form.js]: - [x] Please use arrow functions. This is a project requirement. Arrow functions are also more compact and fit it well with the functional programming paradigm. Update all functions in form.js, score.js & title.js files. 👍🏾 

  - You can re-write  function to an arrow function simply like this
  ```
const form = () => {
}
```
Note that arrow functions have an implicit return when they are oneliners

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Dachrono , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All Linter checks are passing ✔️ 
- [ ] Atomic commit messages ✔
- [ ]  Correct Git flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/form.js (hunk 1 above)
Comment (Dachrono) [reply src/modules/form.js]: tky for your comments and your patience, I change all functions for arrow functions

**Review by wayungi — APPROVED** [review]
Hi @Dachrono , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Profesional README ✔️ 
- [ ] Used arrow functions ✔️  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**